### ISSUE TITLE
error (payload): error: cannot connect to mongodb. details: querytxt etimeout

### ISSUE THREAD

**Issue by adekmo** [adrianhajdin/ecommerce#9]
please help me to resolve this issue.
I've tried following this solution
https://stackoverflow.com/questions/61800062/error-querytxt-etimeout-when-connecting-to-mongodb-atlas-using-mongoose
but it's still not working.

**Comment by ahmadsohail404** [comment]
did u fix the issue? @adekmo

**Comment by adekmo** [comment]
yes i did, i dont know why but i only change the version mongodb to 2.2.12 or later when choose connect to cluster with driver.

**Comment by sohailatwork** [comment]
I simply fixed it by running: networksetup -setdnsservers Wi-Fi 8.8.8.8 8.8.4.4

**Comment by ahmadsohail404** [comment]
I fixed it simply by running: networksetup -setdnsservers Wi-Fi 8.8.8.8 8.8.4.4

**Comment by UmairSaqibBhutta** [comment]
One reason for this error is using mobile hotspot. if we change it to wifi then error will be fixed.
